examples/remarketing/upload_conversion_adjustment.py

```diff
@@ -0,0 +1,124 @@
+    else:
+        raise ValueError('Invalid adjustment type specified.')
+
+    # Create the conversion adjustment and set basic values.
+    conversion_adjustment = (client.get_type('ConversionAdjustment',
+                                             version='v2'))
+    conversion_action_service = (client.get_service('ConversionActionService',
+                                                    version='v2'))
+    conversion_adjustment.conversion_action.value = (
+        conversion_action_service.conversion_action_path(
+            customer_id, conversion_action_id)
+    )
+    conversion_adjustment.adjustment_type = conversion_adjustment_type
+    conversion_adjustment.adjustment_date_time.value = adjustment_time
+
+    # If the restatement value is specified, set it (default currency is USD).
+    if (restatement_value and
+        conversion_adjustment_type ==
+        conversion_adjustment_type_enum.RESTATEMENT):
+        conversion_adjustment.restatement_value.adjusted_value.value = (
+            float(restatement_value))
+        conversion_adjustment.restatement_value.currency_code.value = 'USD'
+
+    # Set the Gclid Date
+    conversion_adjustment.gclid_date_time_pair.gclid.value = gcl_id
+    conversion_adjustment.gclid_date_time_pair.conversion_date_time.value = (
+        conversion_time)
+
+    # Try the upload
+    conversion_adjustment_upload_service = (
+        client.get_service('ConversionAdjustmentUploadService', version='v2')
+    )
+
+    try:
+        response = (
+            conversion_adjustment_upload_service.
+            upload_conversion_adjustments(customer_id,
+                                          [conversion_adjustment],
+                                          partial_failure=True)
+        )
+        conversion_adjustment_result = response.results[0]
+        print(f'Uploaded conversion that occurred at '
+              f'"{conversion_adjustment_result.adjustment_date_time.value}" '
+              f'from Gclid '
+              f'"{conversion_adjustment_result.gclid_date_time_pair.gclid.value}"'
+              f' to "{conversion_adjustment_result.conversion_action.value}"')
+
+    except GoogleAdsException as ex:
+        print(f'Request with ID "{ex.request_id}" failed with status '
+              f'"{ex.error.code().name}" and includes the following errors:')
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f'\t\tOn field: {field_path_element.field_name}')
+        sys.exit(1)
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description='Upload an offline conversion.')
```

Comment:
The  description is wrong, it should be "Uploads a conversion adjustment"

Reply:
fixed